app/src/test/java/com/kickstarter/viewmodels/FacebookConfimationViewModelTest.java

```diff
@@ -98,12 +98,12 @@ public void testToggleSendNewsLetter_ForUSUser() {
     this.vm = new FacebookConfirmationViewModel.ViewModel(environment);
 
     this.vm.outputs.sendNewslettersIsChecked().subscribe(this.sendNewslettersIsChecked);
-    this.sendNewslettersIsChecked.assertValue(true);
+    this.sendNewslettersIsChecked.assertValue(false);
```

Comment:
I would change the name of the test to something like `testToggleSendNewsLetter_isNotChecked`, since now we don't differenciate the toggle status for US/Non-US users.